scripts/meetingbaas.py

```
@@ -413,11 +414,32 @@ async def queue_initial_message():
+            # The persona_data should contain a key that matches the folder name
+            # For now, we'll extract it from the persona data or use a mapping
+            # The persona folder names are like "interviewer", not "Technical Interviewer Bot"
+            # We need to find the folder name that corresponds to this persona
+            from config.persona_utils import PersonaManager
+            pm = PersonaManager()
+            # Try to find persona by matching the display name
+            for folder_name, data in pm.personas.items():
+                if data.get("name") == persona_data.get("name"):
+                    persona_name = folder_name
+                    break
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)